md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/PropertiesTemplateAPI.java

```diff
@@ -44,27 +43,25 @@
      * corresponding to processId passed.
      *
      * @param processTemplateId
-     * @param model
      * @return nothing.
      */
     @Autowired
     PropertiesTemplateDAO propertiesTemplateDAO;
 
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
-    public
+
     @ResponseBody
-    RestWrapper delete(@PathVariable("id") Integer processTemplateId, Principal principal,
-                       ModelMap model) {
+    public RestWrapper delete(@PathVariable("id") Integer processTemplateId, Principal principal) {
 
         RestWrapper restWrapper = null;
         try {
-            // s.delete("call_procedures.DeletePropertiesTemplate", propertiesTemplate);
             propertiesTemplateDAO.deletePropertiesTemplate(processTemplateId);
 
             restWrapper = new RestWrapper(null, RestWrapper.OK);
             LOGGER.info("Record with ID:" + processTemplateId + " deleted from PropertiesTemplate by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)